.cspell.json

```diff
@@ -444,6 +444,7 @@
     "datadir",
     "dataend",
     "datafiles",
+    "dataframe",
```

Comment:
I think this is usually written as `DataFrame`, which doesn't cause an error.

```suggestion
```

Reply:
Sorry, but I failed in checking it. This must be excluded in this PR.

@bopeng-saitama  Could you replace dataframe by another commonly-used word in CARET's source code.

Reply:
@takam5f2 Noted on this. I'll do it right away.